website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -30,6 +30,45 @@ Notice here how the OAuth Scopes are grouped by the two fields `view` and `manag
 
 This grouping determines the **mapping and relation between OAuth Scopes and user permissions**.
 
+# Permission groups
+
+<Info>
+
+This feature is available from version `21.21.0` onwards.
+
+</Info>
+
+Defining `oAuthScopes` in the Custom Application config yields the default permision group which allows using permissions limited to 1 unique pair (view/manage) specific to the Custom Application. The default permission pair can cover access control needs of Custom Applications intended to be used by one team.
+
+However, in certain cases more granular access control is needed. For example, if multiple teams accross the organization have access to a Custom Application but only one team is allowed to manage discount codes, the default permission pair is not enough.
+
+For more granular permissions [additional OAuth Scopes](https://docs.commercetools.com/api/scopes) can be requested as part of various additional permission groups. 
+
+These additional OAuth Scopes must be specified in your Custom Application config, using the [`additionalOAuthScopes` field](/api-reference/application-config#additionaloauthscopes).
+
+In the following example, permission group named `team-a` allows to manage orders but not manage discount codes, while permission group named `team-b` allows both.
+
+```json title="custom-application-config.json" highlightLines="6-17"
+{
+  "oAuthScopes": {
+    "view": ["view_products", "view_customers"],
+    "manage": ["manage_products"]
+  },
+  "additionalOAuthScopes": [
+    {
+      "name": "team-a",
+      "view": [],
+      "manage": ["manage_orders"]
+    },
+    {
+      "name": "team-b",
+      "view": [],
+      "manage": ["manage_orders", "manage_discount_codes"]
+    }
+  ]
+}
+```
+
```

Comment:
```suggestion

<Info>

The default permission group is always defined, even when adding additional groups.

When additional groups are defined, the default group can be left "empty", without specifying any OAuth Scopes.

Even so, remember that the "view-only" user permission must be enabled to allow access to the Custom Application.

</Info>

```

Reply:
💎 
Just to double check:

> The default permission group is always defined, even when **not** adding additional groups

Reply:
Hmm I believe is correct the way it is.

How I read it:

1. `The default permission group is always defined` implies that this is a fact and won't change.
2. `even when adding additional groups` implies that the previous statement `always defined` will continue to be true.

To counter the first statement you would use something like `except`.

Does that make sense 🤔

Reply:
Thanks for explaining, I get your point now 😄